src/main/java/uk/gov/hmcts/darts/common/repository/EventHandlerRepository.java

```diff
@@ -22,14 +22,12 @@ public interface EventHandlerRepository extends JpaRepository<EventHandlerEntity
     List<EventHandlerEntity> findByTypeAndSubType(String type, String subType);
 
     @Query("""
-        SELECT EXISTS (
-            SELECT 1 FROM EventHandlerEntity eh
-            WHERE eh.type = :type
-            AND eh.subType = :subType
-            AND eh.active = true
-        )
+        SELECT eh FROM EventHandlerEntity eh
+        WHERE eh.type = :type
+        AND eh.subType = :subType
+        AND eh.active = true
         """)
-    boolean doesActiveMappingForTypeAndSubtypeExist(String type, String subType);
+    List<EventHandlerEntity> findActiveMappingsForTypeAndSubtypeExist(String type, String subType);
```

Comment:
Would it be better to drop the `Exist` part of the method name: i.e `findActiveMappingsForTypeAndSubtype`?